src/fluree/db/flake.cljc

```diff
@@ -549,6 +499,15 @@
   [ss to-remove]
   (transient-reduce disj! ss to-remove))
 
+(defn revise
+  "Changes the composition of the sorted set `ss` by adding all the flakes in the
+  `to-add` collection and removing all flakes in the `to-remove` collection."
+  [ss to-add to-remove]
+  (as-> (transient ss) trans
+    (reduce disj! trans to-remove)
+    (reduce conj! trans to-add)
```

Comment:
Does `reduce` negate some of the benefit of the transient here? Since we don't care about the return value each time, would it be noticeably more performant to just loop these for the side effects?

Reply:
You're totally right! I just didn't think about it. I'll change it.